### PR TITLE
fix(Input): should not have value prop on input after click toggle icon

### PR DESCRIPTION
**Pull request by linxianxi**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Fix Input.Password that should not have value prop on input after click toggle icon          |
| 🇨🇳 Chinese |  修复 Input.Password 在点击隐藏按钮后 input 上会有 value 属性的问题         |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/input/hooks/useRemovePasswordTimeout.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { useEffect, useRef } from 'react';
+import type { InputRef } from '../Input';
+
+export default function useRemovePasswordTimeout(
+  inputRef: React.RefObject<InputRef>,
+  triggerByIcon?: boolean,
```
Comment (MadCcc) [inline components/input/hooks/useRemovePasswordTimeout.ts:6]: 这个参数看起来有点怪，如果只是想控制 mount 时是否执行的话应该换个名字

Comment (linxianxi) [reply components/input/hooks/useRemovePasswordTimeout.ts:6]: 换什么名字比较好？

File: components/input/hooks/useRemovePasswordTimeout.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+          inputRef.current?.input.removeAttribute('value');
+        }
+      }),
+    );
+  };
+
+  useEffect(() => {
+    if (!triggerOnMount) {
+      removePasswordTimeout();
+    }
```
Comment (MadCcc) [inline components/input/hooks/useRemovePasswordTimeout.ts:26]: 意思是不是反了

Comment (linxianxi) [reply components/input/hooks/useRemovePasswordTimeout.ts]: 嗯 忘改了

**Review by MadCcc — APPROVED** [review]

**Pull request merged**